### PR TITLE
added release notes v0.4.3

### PR DESCRIPTION
**Pull request by anshdhinhgra47**
Signed-off-by: Ansh Dhingra  anshdhingra47@gmail.com

This PR fixes ##1598

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/project/releases.md [hunk 1]
```diff
@@ -61,7 +61,25 @@ permalink: project/releases
 
 # Meshery Releases
 
+### [v0.4.3](https://github.com/layer5io/meshery/releases/tag/v0.4.3)
+Released on 22 August
```
Comment (kushthedude) [inline docs/pages/project/releases.md:65]: Can you please add year also?

File: docs/pages/project/releases.md [hunk 2]
```diff
@@ -61,7 +61,25 @@ permalink: project/releases
...
+  + Doc pages fixed
+  + Updated README.md
+  + Added first timers' and welcome guide link
+  + Added "Open Service Mesh" to navigation menu in Meshery docs
+
+- Other fixes
+  + smi-conformance - added Colons to front matter
+  + Missing bottom row added
 ### [v0.4.1](https://github.com/layer5io/meshery/releases/tag/v0.4.1)
 Released on 8 August
```
Comment (kushthedude) [inline docs/pages/project/releases.md:84]: Please add year here too

**Review by kushthedude — CHANGES_REQUESTED** [review]

**Comment by anshdhinhgra47** [comment]
Should I then create a new PR afterwards ?

**Comment by kushthedude** [comment]
> Should I then create a new PR afterwards ?

The following change can be accommodated in the same PR, having more micro-PRs can lead to a bad git commit history.

**Comment by anshdhinhgra47** [comment]
Oh okay

**Comment by anshdhinhgra47** [comment]
I'll just add

**Comment by kushthedude** [comment]
Thank you !

**Comment by anshdhinhgra47** [comment]
Added year to the dates. Please review

File: docs/pages/project/releases.md [hunk 3]
```diff
@@ -61,9 +61,27 @@ permalink: project/releases
 
 # Meshery Releases
 
+### [v0.4.3](https://github.com/layer5io/meshery/releases/tag/v0.4.3)
+Released on 22 August 2020
 
+**What's new ?**
+- Featured
```
Comment (kushthedude) [inline docs/pages/project/releases.md:68]:
```suggestion
- Features
```

File: docs/pages/project/releases.md [hunk 4]
```diff
@@ -61,9 +61,27 @@ permalink: project/releases
...
+  + Added tooltips for navigation menu items while in minimised state 
+  + Added community calendar button
+
+- Documentation
+  + Doc pages fixed
+  + Updated README.md
+  + Added first timers' and welcome guide link
+  + Added "Open Service Mesh" to navigation menu in Meshery docs
+
+- Other fixes
```
Comment (kushthedude) [inline docs/pages/project/releases.md:80]:
```suggestion
- Bug fixes
```

**Review by kushthedude — APPROVED** [review]
Thank you @anshdhinhgra47, great work here!

**Comment by KarenEfereyan** [comment]
@anshdhinhgra47 well done

**Comment by anshdhinhgra47** [comment]
Thank you so much 😊

**Pull request merged**